pkg/syncer/spec/spec_controller.go

```diff
@@ -96,7 +99,47 @@ func NewSpecSyncer(gvrs []schema.GroupVersionResource, upstreamClusterName logic
 				c.AddToQueue(gvr, obj)
 			},
 		})
-		klog.InfoS("Set up informer", "clusterName", upstreamClusterName, "pcluster", workloadClusterName, "gvr", gvr.String())
+		klog.V(2).InfoS("Set up upstream informer", "clusterName", upstreamClusterName, "pcluster", workloadClusterName, "gvr", gvr.String())
+
+		downstreamInformers.ForResource(gvr).Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+			DeleteFunc: func(obj interface{}) {
+				unstrob, ok := obj.(*unstructured.Unstructured)
```

Comment:
this is not good. I did the same mistake here: https://github.com/kcp-dev/kcp/pull/1049. 
Background: if the informer restarts in just this moment, it can be that no old object is return, but some tombstone object. You have to use the key helper like in upper PR.

Reply:
@sttts see https://github.com/kcp-dev/kcp/pull/1046#discussion_r872716552